zcbor/zcbor.py

```diff
@@ -2599,7 +2599,13 @@ def parse_args():
     code_parser.add_argument(
         "--output-cmake", required=False, type=str,
         help="""Path to output Cmake file. The filename of the Cmake file without '.cmake' is used
-as the name of the Cmake target in the file.""")
+as the name of the Cmake target in the file.
```

Comment:
nitty nit nitteson: I guess we should use either 'cmake' or 'CMake', not 'Cmake'